d2.config.js

```diff
@@ -20,6 +20,8 @@ const config = {
     },
 
     coreApp: true,
+
+    dataStoreNamespace: 'DHIS_WEB_MAPS',
```

Comment:
I'm thinking we could make this namespace even more unique to avoid potential conflicts.

Reply:
fix: [455af79](https://github.com/dhis2/maps-app/pull/3198/commits/455af79d46a6b88a0fcb6716db15de1d4f31c068)